ptp/multipagepdf.py

```diff
@@ -4,49 +4,80 @@
 from pathlib import Path
 import sys
 import subprocess
-import traceback
 
 from ocrd_models import OcrdMets
+from ocrd_utils.logging import getLogger
+from atomicwrites import atomic_write
+from ocrd_models.constants import NAMESPACES as NS
 
-def read_from_mets(metsfile, outputfile):
+def get_metadata(mets):
+    title = mets._tree.getroot().find('.//mods:title', NS)
+    subtitle = mets._tree.getroot().find('.//mods:subtitle', NS)
+    title = title.text if title is not None else ""
+    title += "Subtitle: "+subtitle.text if subtitle else ""
+    publisher = mets._tree.getroot().find('.//mods:publisher', NS)
+    author = mets._tree.getroot().find('.//mods:creator', NS)
+    return {'Author':author.text if author is not None else "",
+            'Title': title,
+            'Keywords': publisher.text+" (Publisher)" if publisher is not None else ""}
```

Comment:
Nice!

@kba @tboenig Is this already comparable to what DFGViewer provides in its PDF export?